Tests/Opc.Ua.Client.Tests/ClientFixture.cs

```diff
@@ -343,6 +352,34 @@ private void Session_KeepAlive(ISession session, KeepAliveEventArgs e)
                 session?.Dispose();
             }
         }
+
+        private void ConfigureActivityListener()
+        {
+            // Set the default ID format to W3C and force it to be the default
+            Activity.DefaultIdFormat = ActivityIdFormat.W3C;
+            Activity.ForceDefaultIdFormat = true;
+
+            // Create an instance of ActivityListener and configure its properties
+            ActivityListener activityListener = new ActivityListener()
+            {
+                // Set ShouldListenTo property to true for all activity sources
+                ShouldListenTo = (source) => true,
+
+                // Sample all data and recorded activities
+                Sample = (ref ActivityCreationOptions<ActivityContext> options) => ActivitySamplingResult.AllDataAndRecorded,
+
+                // Write "Started" message when an activity starts
+                ActivityStarted = activity => Console.WriteLine("Started: {0,-15} {1,-60}", activity.OperationName, activity.Id),
+
+                // Write "Stopped" message along with OperationName, Id, and Duration when an activity stops
+                ActivityStopped = activity =>
+                {
+                   Console.WriteLine(activity.OperationName + " : " + activity.Id + "Duration : " + activity.Duration);
+                }
+            };
+
+            ActivitySource.AddActivityListener(activityListener);
```

Comment:
This method looks like debug code, let's remove it before we submit this part.

Reply:
Added cleanup method for Activity listener

Reply:
At this time I'm ok to have a listener here since this is just test code and activity source is only supported with .NET6 and greater. Ideally the listener is better integrated in the test setup and is optional to use. Since now its mostly about the test coverage, it fulfills the purpose.